openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -104,15 +104,23 @@ const EntityTable = ({
   const fetchTagsAndGlossaryTerms = () => {
     setIsTagLoading(true);
     Promise.allSettled([getClassifications(), fetchGlossaryTerms()])
-      .then((values) => {
+      .then(async (values) => {
         let tagsAndTerms: TagOption[] = [];
         if (
           values[0].status === SettledStatus.FULFILLED &&
           values[0].value.data
         ) {
-          tagsAndTerms = getTaglist(values[0].value.data).map((tag) => {
-            return { fqn: tag, source: 'Tag' };
-          });
+          const tagList = await getTaglist(values[0].value.data);
+
+          tagsAndTerms =
+            tagList.length !== 0
+              ? tagList.map((tag) => {
+                  return {
+                    fqn: tag,
+                    source: 'Tag',
+                  };
+                })
+              : [];
```

Comment:
same here

Reply:
same as above, We can keep note of this and probably update it all together with proper approach